be/src/runtime/runtime_state.cpp

```diff
@@ -467,4 +467,38 @@ int64_t RuntimeState::get_load_mem_limit() {
     }
 }
 
+void RuntimeState::set_global_dicts(const std::shared_ptr<TGlobalDict>& tglobal_dict) {
+    if (tglobal_dict) {
+        for (auto it = tglobal_dict->dicts.begin(); it != tglobal_dict->dicts.end(); it++) {
+            _dict_id_to_global_dict_map[it->first] =
+                    std::make_shared<vectorized::GlobalDict>(it->second.str_dict);
+        }
+        for (auto it = tglobal_dict->slot_dicts.begin(); it != tglobal_dict->slot_dicts.end();
+             it++) {
+            assert(_dict_id_to_global_dict_map.find(it->second) !=
+                   _dict_id_to_global_dict_map.end());
+            _slot_id_to_global_dict_map[it->first] = _dict_id_to_global_dict_map[it->second];
+        }
+    }
+}
+
+vectorized::GlobalDictSPtr RuntimeState::get_global_dict(int slot_id) {
+    assert(_slot_id_to_global_dict_map.find(slot_id) != _slot_id_to_global_dict_map.end());
+    return _slot_id_to_global_dict_map[slot_id];
+}
+
+vectorized::GlobalDictSPtr RuntimeState::find_global_dict(int slot_id) {
```

Comment:
find global dict is duplicated with get_global_dict please merge these 2 method to 1